src/custom/utils/index.ts

```diff
@@ -17,79 +17,45 @@ export {
   formattedFeeAmount,
 } from '@src/utils'
 
-const ETHERSCAN_PREFIXES: { [chainId in ChainId]: string } = {
-  1: '',
-  // 3: 'ropsten.',
-  4: 'rinkeby.',
-  5: 'goerli.',
-  // 42: 'kovan.',
-  100: 'xdai.',
+const ETHERSCAN_URLS: { [chainId in ChainId]: { prefix: string; domain: string } } = {
```

Comment:
Re-done this constant to add also the domain, since it's different for gchain than all others